go/client/commands_common.go

```diff
@@ -25,6 +25,7 @@ func GetCommands(cl *libcmdline.CommandLine, g *libkb.GlobalContext) []cli.Comma
 		NewCmdDb(cl, g),
 		NewCmdDevice(cl, g),
 		NewCmdID(cl, g),
+		NewCmdInstall(cl, g),
```

Comment:
Fixed, created getPlatformSpecificCommands as you suggested with commands_osx and commands_non_osx